Client/Application/Migration.swift

```diff
@@ -47,6 +47,7 @@ fileprivate extension Preferences {
         migrate(key: "blockPopups", to: Preferences.General.blockPopups)
         migrate(key: "kPrefKeyTabsBarShowPolicy", to: Preferences.General.tabBarVisibility)
         migrate(key: "NightModeStatus", to: Preferences.General.nightMode)
+        migrate(key: "allowBackgroundMediaPlayback", to: Preferences.General.allowBackgroundMediaPlayback)
```

Comment:
Migration keys were added for preferences in older versions of Brave.
Since this is a new preference, we don't need to migrate it